src/screens/LocationPage/LocationPage.js

```diff
@@ -10,6 +11,30 @@ import EnsureSharingIdInUrl from 'components/EnsureSharingIdInUrl';
 import ChartsHolder from 'components/LocationPage/ChartsHolder';
 import { LoadingScreen } from './LocationPage.style';
 import { useProjections } from 'common/utils/model';
+import { findLocationForFips, findStateByFips } from 'common/locations';
+import { LOCATION_SUMMARY_LEVELS } from 'common/metrics/location_summary';
+
+function getPageTitle(projections) {
+  const location = findLocationForFips(projections.fips);
+  const state = findStateByFips(location.state_fips_code);
+  const locationName = location.county
+    ? `${location.county}, ${state.state}`
+    : `${location.state}`;
+  return `${locationName}, ${location.state_code} - America’s COVID Warning System`;
```

Comment:
We're using `state.state`, `location.state`, and `location.state_code` ...  is this really necessary / correct?  As-is, it's confusing.  If it's really necessary maybe add some intermediate variables (`stateName`, `stateCode`, etc.) to capture the distinction between these values?